pkg/orchestration/wiring/wiringutil/osb/service_instance_wiring.go

```diff
@@ -0,0 +1,72 @@
+package osb
```

Comment:
This code is based on https://github.com/atlassian/voyager/blob/master/pkg/orchestration/wiring/wiringutil/svccatentangler/plugin.go, but re-writen using a set of helper functions rather than a whole wiring plugin implementation with callbacks

Reply:
One thing to watch out for is that helper libraries can grow out of control pretty quickly :-/

Reply:
@brjacobsatlassian at the moment it's opposite (see original code), but we should keep an eye on this code, of course.